packages/flutter/lib/src/material/mergeable_material.dart

```diff
@@ -234,6 +238,20 @@ class _MergeableMaterialState extends State<MergeableMaterial> {
     return false;
   }
 
+  void _removeEmptyGaps() {
+    int j = 0;
+
+    while (j < _children.length) {
+      if (_children[j] is MaterialGap &&
+          _animationTuples[_children[j].key].controller.status
+          == AnimationStatus.dismissed) {
```

Comment:
please put the == on the end of the previous line and indent this one by 2 (or put it all on one line)